run_game.py

```diff
@@ -48,12 +50,18 @@
             WORDS.extend(words)
         print(f"Words we will use for the game: {sorted(WORDS)[:10]}")
 
-        data = get_players()
-
-        players = [
-            PlayerDefinition(row["Team name"], RemotePlayer(row["Team IP or URL (with port if necessary)"]))
-            for i, row in data.iterrows()
-        ]
+        if GAME_SCOPE == "GLOBAL":
+            from server.players import get_global_players
+            players = get_global_players()
+        else:
+            players = []  # [...manually defined list...]
+            # Example:
+            fasttext_model = fasttext.load_model("models/2021_06_05_processed.model")
+            player = LocalFasttextPlayer(model=fasttext_model)
+            players = [
+                PlayerDefinition('HerokuOrg team', RemotePlayer('https://obscure-everglades-02893.herokuapp.com')),
+                # PlayerDefinition('Your trained remote player', RemotePlayer('http://35.246.139.13/')),
+                PlayerDefinition('Local Player', player)]
```

Comment:
Done
It doesn't pass `flake` at this point:
```./run_game.py:17:1: F401 'the_hat_game.players.LocalFasttextPlayer' imported but unused ```

But I think it makes sense to leave LocalFastTextPlayer there, so that students could see where this `LocalFasttextPlayer` could be used.